### PR TITLE
Adding Question Javascript - eventListener

### PR DESCRIPTION
**Pull request by toukirkhan**
Ref to issue #14. Added one quiz question about the addEventListener() method based on Event Handling and another question about Type coercion in Javascript.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/javascript-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,22 @@
 const javascriptQuiz = [
+  {
+    Question: "The process of converting value from one type to another (such as string to number, object to boolean, and so on) is known as?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:3]:
```suggestion
    Question: "What is the process of converting a value from one data type to another called?",
```

File: src/data/javascript-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,22 @@
...
+    Answer: "Type Coercion",
+    Distractor1: "Type Casting",
+    Distractor2: "Data Converting",
+    Distractor3: "Change Value",
+    Explanation: "Type coercion is the process of converting value from one type to another (such as string to number, object to boolean, and so on).",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:8]:
```suggestion
    Explanation: "Type coercion is the process of converting a value from one type to another (such as a string to number, object to boolean, and so on).",
```

File: src/data/javascript-quiz.ts [hunk 3]
```diff
@@ -1,4 +1,22 @@
...
+    Link: "https://www.freecodecamp.org/news/js-type-coercion-explained-27ba3d9a2839/",
+  },
+  {
+    Question: "Which of the following method allows you to set up functions to be called when a specified event happens, such as when a user clicks a button?",
```
Comment (jdwilkin4) [inline src/data/javascript-quiz.ts:12]:
```suggestion
    Question: "Which of the following methods allows you to set up functions to be called when a specified event happens, such as when a user clicks a button?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by toukirkhan** [comment]
@jdwilkin4 I have done the requested changes

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by toukirkhan** [comment]
Thanks will definitely do more contributions.

**Pull request merged**